### PR TITLE
Create license.md

### PR DESCRIPTION
**Pull request by jedwards-habana**
Add license.md file.

### CODE REVIEW

**Comment by jedwards-habana** [comment]
Used addlicense go utility to add license headers to all *.sh, *.py, *.yaml and *.ipynb files. Command used:

'addlicense -c 'Intel Corporation' -y 2022 -l MIT -s ./**/*.py ./**/*.sh ./**/*.yaml ./**/*.ipynb'

**Review by sabinastanescu — CHANGES_REQUESTED** [review]
@jedwards-habana should the license.md also include the MIT license license identifier/

**Comment by jedwards-habana** [comment]
> @jedwards-habana should the license.md also include the MIT license license identifier? I see it in the text in the headers.

File: monday_connector/flattening.py [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# SPDX-License-Identifier: MIT
```
Comment (jedwards-habana) [inline monday_connector/flattening.py:20]: @sabinastanescu Is this what you wanted in the license.md file?

File: license.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ 
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+OR OTHER DEALINGS IN THE SOFTWARE.
+
+SPDX-License-Identifier: MIT
```
Comment (jedwards-habana) [inline license.md:21]: @sabinastanescu The SPDX-License-Identifier is already in license.md. Is that what you want.

Comment (sabinastanescu) [reply license.md:21]: yes, looks good!

**Review by sabinastanescu — APPROVED** [review]
Looks like we have a license.md file + license reference for MIT as the header in every script file.

**Pull request merged**